js/tests/unit/carousel.js

```diff
@@ -989,4 +1018,189 @@ $(function () {
       }, 80)
     }, 80)
   })
+
+  QUnit.test('should allow swiperight and call prev with pointer events', function (assert) {
```

Comment:
does this only simulate/test touch events? or does it also test if the code works correctly for pointer events?